README.md

```diff
@@ -1,4 +1,4 @@
-# MDTF-diagnostics [![Build Status](https://travis-ci.org/tsjackson-noaa/MDTF-diagnostics.svg?branch=develop)](https://travis-ci.org/tsjackson-noaa/MDTF-diagnostics)
```

Comment:
New PODs shouldn't modify top-level repo files. In this case, it appears the changes are showing up because you created your pod/seaicesuite branch from the NOAA-GFDL /main branch, but our policy for new PODs is to merge the new code into /develop. Inconsistency between /main and /develop is our fault (issue #135).